.envrc

```diff
@@ -0,0 +1,27 @@
+# This file is used by direnv to setup the environment when entering to use fvm's flutter.
+
+# Comment out the next line if you want to use your system flutter.
+PATH_add .fvm/flutter_sdk/bin
+
+# Check flutter version matches what's in .fvmrc
+check_flutter_version() {
+  local fvm_flutter_version_string=$(cat .fvmrc)
+  # Fetch string from format `"flutter": "18340ea16c"``
+  local fvm_flutter_version=$(echo "$fvm_flutter_version_string" | grep -o '"flutter": "[a-z0-9]*"' | cut -d '"' -f 4)
+
+  local flutter_version_string=$(flutter --version)
```

Comment:
Sadly `flutter --version` is fairly slow — which makes this take around 1.5 seconds when I `cd` into this directory. That'd get pretty annoying.

To handle that, see the workaround in `print_header` in `tools/check`. For any code there which you'd like to reuse, please feel free to move functions to a file in `tools/lib/` (and/or an existing file there like `tools/lib/git.sh`).